rfid_measurements_filelist: ["/home/relief-user0/catkin_ws/src/relief-rfid-detection/application/results_001625127C5A.txt"]
